admiral/pkg/clusters/virtualservice_routing.go

```diff
@@ -514,10 +515,10 @@ func generateVirtualServiceForIncluster(
 	virtualService.Name = fmt.Sprintf("%s-%s", vsName, inclusterVSNameSuffix)
 
 	// Add the exportTo namespaces to the virtual service
-	if common.EnableExportTo(vsName) {
-		exportToNamespaces := getSortedDependentNamespaces(
+	virtualService.Spec.ExportTo = []string{common.GetSyncNamespace()}
+	if common.EnableExportTo(vsName) && common.DoVSRoutingInClusterForClusterAndIdentity(sourceCluster, sourceIdentity) {
```

Comment:
hey @vrushalijoshi , can we keep the info logs for now for both enabled and disabled cluster/identities? It will help us debug. We'll later move it to a debug log

Reply:
sure, will add it